packages/earth-admin/src/components/detail-list/DetailList.tsx

```diff
@@ -20,17 +20,26 @@
 import * as React from 'react';
 
 import { LinkWithOrg } from 'components/link-with-org';
-import { PlaceIntersectionProps } from 'pages-client/places/model';
+import classNames from 'classnames';
+
+interface DetailListProps {
+  data: any,
```

Comment:
`data` should have stricter typing
```suggestion
interface ISomething {
  id: string
  name: string
}
  data: [ISomething],
```